README.md

```diff
@@ -4,41 +4,57 @@
 
 > This project aims to create toolkit of tools and documentation for generating test data sets based on admin-selected criteria.
 
+> This project a proud part of Salesforce.org Open Source Commons initiative.
```

Comment:
Can we put a link to OSC here? Or a one-sentence description of what Open Source Commons is?  (I spent a few minutes searching for a good page to link to but did not quickly find a CONCISE definition... "Open Source Commons is a community of mission-driven individuals working together to solve the world's most challenging problems" is a little vague...)

Reply:
I don't think we have a link but I'm sure @coriobriensfdo can suggest better language from boiler plate she already has!